deeplake/api/dataset.py

```diff
@@ -1273,6 +1292,7 @@ def ingest_yolo(
     def ingest_classification(
         src: Union[str, pathlib.Path],
         dest: Union[str, pathlib.Path],
+        sample_compression: str = "auto",
```

Comment:
@aadityasinha-dotcom This parameter should be deleted, because sample_compression should be specified in the image_params (should be renamed label_params). If sample_compression is not specified there, it should default to auto.